### DISCUSSION TITLE
Can't debug quasar project

### DISCUSSION THREAD

**Discussion by 2BJulien** [quasarframework/quasar#9174] [CLI - PWA mode]
Every time I try to debug a file in vs code, it crashes instantly. I don't know why this is happening, but I suspect it has something to do with corejs.

That is the error I get
![2021-05-05 12_30_39-Window](https://user-images.githubusercontent.com/71118357/117129364-df6f8d80-ad9e-11eb-8af3-57e0945b1fc0.png)

The file in which the error occurred
![2021-05-05 12_40_13-Window](https://user-images.githubusercontent.com/71118357/117129530-15147680-ad9f-11eb-876b-0bbc3d242fef.png) 

I have already reinstalled my node_modules and tried the suggestion on this site: https://forum.quasar-framework.org/topic/5146/solved-core-js-issue/12. But none of these helped me with my problem.

Thanks in advance!

**Comment by hawkeye64** [comment]
@2BJulien You haven't specified what version of quasar. Different versions may require different remedies.

**Reply by 2BJulien** [reply]
My bad, sorry! I'm using version 3.0.0-beta.18.

**Reply by hawkeye64** [reply]
You are referring to @quasar/app, so I presume your quasar version is v2.0.0-beta.14.

In your packages.json, can you show the version of core-js?

Or, run the following:

```bash
$ yarn why core-js
yarn why v1.22.5
[1/4] Why do we have the module "core-js"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "core-js@3.11.1"
```

**Reply by 2BJulien** [reply]
Here my package.json's dependencies:
```
 "dependencies": {
    "@quasar/extras": "^1.0.0",
    "axios": "^0.21.1",
    "core-js": "^3.11.2",
    "eslint": "^7.25.0",
    "quasar": "^2.0.0-beta.1",
    "vue-i18n": "^9.0.0-beta.0"
  },
  "devDependencies": {
    "@quasar/app": "^3.0.0-beta.1"
  },
```

**Reply by hawkeye64** [reply]
What's your platform?

**Reply by 2BJulien** [reply]
Windows

**Reply by hawkeye64** [reply]
Can you roll back to core-js v3.9.1 and see how that goes for you? Don't forget to delete node_modules and yarn.lock

**Reply by 2BJulien** [reply]
Thanks! I will give it a shot.

**Reply by 2BJulien** [reply]
I am still getting the error.

**Reply by hawkeye64** [reply]
Are you using ArrayBuffer anywhere in your code?

**Reply by 2BJulien** [reply]
Sorry for the late reply, I had a day off from work.  No, I don't use Arraybuffer.

**Reply by hawkeye64** [reply]
yarn or npm and what versions? Can you try `yarn why core-js` or `npm why core-js` (I think you need v7.x of NPM for this command).

**Reply by 2BJulien** [reply]
core-js@3.9.1
node_modules/core-js
  core-js@"^3.9.1" from the root project

core-js@3.11.2 dev optional
node_modules/global-agent/node_modules/core-js
  core-js@"^3.6.5" from global-agent@2.2.0
  node_modules/global-agent
    optional global-agent@"^2.0.2" from @electron/get@1.12.4
    node_modules/@electron/get
      @electron/get@"^1.0.1" from electron@12.0.6
      node_modules/electron
        dev electron@"^12.0.6" from the root project
      @electron/get@"^1.6.0" from electron-packager@14.2.1
      node_modules/electron-packager
        dev electron-packager@"^14.2.1" from the root project
      @electron/get@"^1.3.1" from @types/electron-packager@14.0.0
      node_modules/@types/electron-packager
        @types/electron-packager@"14.0.0" from @quasar/app@3.0.0-beta.18
        node_modules/@quasar/app
          dev @quasar/app@"^3.0.0-beta.1" from the root project

core-js@3.11.2 dev
node_modules/@quasar/babel-preset-app/node_modules/core-js
  core-js@"^3.6.5" from @quasar/babel-preset-app@2.0.1
  node_modules/@quasar/babel-preset-app
    @quasar/babel-preset-app@"2.0.1" from @quasar/app@3.0.0-beta.18
    node_modules/@quasar/app
      dev @quasar/app@"^3.0.0-beta.1" from the root project

**Reply by hawkeye64** [reply]
It almost looks like, you can remove your local version and the one used by `@quasar/babel-preset-app` will be hoisted. Although, I am not sure why that did not show up for me.

**Reply by 2BJulien** [reply]
Removing core-js just caused more errors and I can't run my program anymore.

**Reply by hawkeye64** [reply]
@2BJulien Sorry, I am at a loss now. :(
As a last resort, I'd say create a new project and slowly port things over.

**Reply by 2BJulien** [reply]
Thanks for the help!
